data/account/partInstall.go

```
@@ -93,22 +97,43 @@ func partMigrate(tx *sql.Tx) (err error) {
 		return ErrUnsupportedSchema
 	}
 
+	partVersion, err = updateDB(tx, partVersion)
+	if err != nil {
+		return err
+	}
+
+	if partVersion != PartTableSchemaVersion {
+		return ErrUnsupportedSchema
+	}
+
+	return
+}
+
+func updateDB(tx *sql.Tx, partVersion int) (int, error) {
 	if partVersion == 1 {
-		_, err = tx.Exec("ALTER TABLE ParticipationAccount ADD keyDilution INTEGER NOT NULL DEFAULT 0")
+		_, err := tx.Exec("ALTER TABLE ParticipationAccount ADD keyDilution INTEGER NOT NULL DEFAULT 0")
 		if err != nil {
-			return
+			return 0, nil
```

We should return the err instead of nil